api/managementpb/checks.proto

```diff
@@ -74,6 +78,9 @@ service SecurityChecks {
       post: "/v1/management/SecurityChecks/Start"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "If no checks names specified all available checks will be started."
```

Comment:
minor: better to say `All the available checks will be started if check names weren't specified.`

Reply:
Done